configuration/wireless/access-point-routed.md

```diff
@@ -0,0 +1,190 @@
+
+### Define the wireless interface IP configuration
+
+In this document, we assume IP network `10.10.0.0/24` is configured on the ethernet LAN, and the Raspberry Pi will manage IP network `192.168.4.0/24` for wireless clients.
+*Note:* Please select another IP network for wireless, e.g. `192.168.10.0/24`, in case IP network `192.168.4.0/24` is already in use by your ethernet LAN.
+
+The Raspberry Pi runs a DHCP server for the wireless network, this requires static IP configuration for the wireless interface (`wlan0`) in the Raspberry Pi. 
+The Raspberry Pi acts as router on the wireless network, as customary we will give it the first IP address in the network: `192.168.4.1`.
+
+To configure the static IP address, edit the configuration file for `dhcpcd` with:
+
+```
+sudo nano /etc/dhcpcd.conf
+```
+
+Go to the end of the file and add the following:
+
+```
+interface wlan0
+    static ip_address=192.168.4.1/24
+    nohook wpa_supplicant
+```
+
+### Enable routing and IP masquerading
+
+This section configures the Raspberry Pi to let wireless clients access computers on the main network, and from there the Internet.
+*NOTE:* If you wish to block wireless clients from accessing the main network and the Internet, skip this section. 
+
+To enable routing, i.e. allow traffic to flow from one network to an other in the Raspberry Pi, create the following file:
+```
+sudo nano /etc/sysctl.d/routed-ap.conf
+
+# https://www.raspberrypi.org/documentation/configuration/wireless/access-point-routed.md
+# Enable IPv4 routing
+net.ipv4.ip_forward=1
+```
+Enabling routing will allow hosts from network `192.168.4.0/24` to reach the LAN and the main router towards the Internet. In order to allow traffic between clients on this foreign wireless network and the Internet without changing the configuration of the main router, the Raspberry Pi can substitute the IP address of wireless clients with its own IP address on the LAN using a "masquerade" firewall rule.
+* The main router will see all outgoing traffic from wireless clients as coming from the Raspberry Pi, allowing communication with the Internet.
+* The Raspberry Pi will receive all incoming traffic, substitute the IP addresses back and forward traffic to the origin wireless client.
+
+This process is configured by adding a single firewall rule in the Raspberry Pi:
+
+```
+sudo iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE
+```
+Now save the current firewall rules for IPv4 (including the rule above) and IPv6 to be loaded at boot by the `netfilter-persistent` service:
+```
+sudo netfilter-persistent save
+```
+Filtering rules are saved to directory `/etc/iptables/`. If in the future you change the configuration of your firewall, make sure to save the configuration to files before rebooting.
```

Comment:
Does this sentence still work if "to files" is removed?

Reply:
It works better :)